database/migrations/20241105192209_public-user-id.sql

```diff
@@ -0,0 +1,9 @@
+-- migrate:up
+ALTER TABLE users
+ADD COLUMN public_id UUID DEFAULT gen_random_uuid() NOT NULL;
```

Comment:
Oh, one more thing: can we add a `UNIQUE` constraint?

Just for brevity. I know UUIDs are not gonna realistically conflict but it's better practice.